guillotina/exceptions.py

```diff
@@ -216,7 +216,15 @@ def json_err_list(self, errors):
             if "error" in error:
                 exc = error.pop("error")
                 if hasattr(exc, "errors") and exc.errors:
-                    error["errors"] = self.json_err_list([e.json() for e in exc.errors])
+                    inner_errors = []
+                    for e in exc.errors:
+                        try:
+                            inner_errors.append(e.json())
+                        except AttributeError:
+                            # jsonschema.exceptions.ValidationError
+                            # does not have json() method
+                            inner_errors.append({"error_str": str(e)})
```

Comment:
not sure what to do here. 
 
I could also do:
```python
if isinstance(jsonschema.AttributeError, e):
   inner_errors.append({"message": e.message})
else:
   inner_errors.append(e.json())
```

What's your opinion?

Reply:
I like this option:

```
if isinstance(jsonschema.exceptions.ValidationError, e):
   inner_errors.append({"message": e.message})
else:
   inner_errors.append(e.json())
```

Reply:
@lferran is this pr still a draft? is ready for review? I'd like to have this fix :)